modules/swagger-codegen/src/main/resources/JavaPlayFramework/newApi.mustache

```diff
@@ -3,11 +3,12 @@ package {{package}};
 {{#imports}}import {{import}};
 {{/imports}}
 
-import play.mvc.Http;
 import java.util.List;
 import java.util.ArrayList;
-import java.util.HashMap;
-import java.io.FileInputStream;
```

Comment:
Same thing here.
For all of these, we could go and make it dynamic by adding some logic in the JavaPlayFrameworkCodegen.java that deals with that but it would take a good amount of work and testing and I currently don't have the time.